examples/twisted/head_request.py

```diff
@@ -94,7 +94,7 @@ def sendRequest(self):
             (':authority', AUTHORITY),
             (':scheme', 'https'),
             (':path', PATH),
-            ('user-agent', 'hyper-h2/1.0.0'),
+            ('user-agent', 'h2/1.0.0'),
```

Comment:
nitpick: this might be a bit confusing now - it's just an example, so not really critical.
Maybe something: `python-hyper/h2/1.0.0`? 
or omit the user-agent entirely?

Reply:
Going to omit it entirely